website/docs/reference/transforms/aws_ec2_metadata.md.erb

```diff
@@ -0,0 +1,23 @@
+<%- component = metadata.transforms.add_tags -%>
```

Comment:
This should be updated to `metadata.transforms.aws_ec2_metadata`. Then you'll need to re-run `make generate`.

Reply:
Done, thanks!